### PR TITLE
task 3254

### PR DESCRIPTION
**Pull request by Airat163**
added sout in methodX

### CODE REVIEW

File: src/main/java/com/example/demo/Batch.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public class Batch {
     public void methodX() {
```
Comment (taron1991) [inline src/main/java/com/example/demo/Batch.java:4]: add parametr List<Integer> and show all elemnets using stream

**Comment by Airat163** [comment]
i have done

File: src/main/java/com/example/demo/Batch.java [hunk 2]
```diff
@@ -1,8 +1,10 @@
 package com.example.demo;
 
-public class Batch {
-    public void methodX() {
+import java.util.List;
 
+public class Batch {
+    public void methodX(List<String> peoples) {
+        peoples.stream().forEach(System.out::println);
     }
```
Comment (taron1991) [inline src/main/java/com/example/demo/Batch.java:8]: change void to List

**Comment by Airat163** [comment]
i changed

**Pull request closed without merging**